R/dependencies.R

```diff
@@ -80,10 +80,10 @@ add_project_to_local_repos <- function(project, local_repos) {
 #' }
 #'
 install_deps <- function(project = ".", feature = NULL,
-                                  local_repos = get_local_pkgs_from_config(),
-                                  direction = "upstream",
-                                  install_project = TRUE, dry_install = FALSE, verbose = 0,
-                                  install_external_deps = TRUE, ...) {
+                         local_repos = get_local_pkgs_from_config(),
+                         direction = "upstream",
```

Comment:
should the direction argument be removed? With the change to `get_true_deps_graph(internal_deps, direction = "upstream")` below `direction` doesn't seem to be used is this right?

Reply:
`direction` is used on line 104. We need the true dependency graph, where arrows point upstream, so we don't pass `direction` here.

Reply:
done, thanks